__tests__/feature/editing/editingLiteral.test.js

```diff
@@ -143,13 +143,33 @@ describe("editing a literal property", () => {
     // Using getByRole here and below because it limits to the visible modal.
     screen.getByRole("heading", { name: "Languages" })
 
+    let radioButtons
+    await waitFor(() => {
```

Comment:
You shouldn't need `waitFor` here. Try `const radioButtons = await screen.findAllByRole("radio")`

Reply:
you are correct, i switched to just `await` and it worked